backend/src/main/java/app/mahmuthoca/controller/AnnotationController.java

```diff
@@ -32,8 +34,13 @@ public AnnotationController(AnnotationService annotationService) {
     }
 
     @GetMapping("/getById")
-    public String getById(@RequestParam("annotationId") Long annotationId) {
-        return annotationService.getById(annotationId);
+    public ResponseEntity<String> getById(@RequestParam("annotationId") Long annotationId) {
+        HttpHeaders responseHeaders = new HttpHeaders();
+        responseHeaders.set("content-type",
+                "application/ld+json;profile=\"http://www.w3.org/ns/anno.jsonld\"");
```

Comment:
To comply with the standards of w3 annotation data model, we added the related content-type header.
> The media type of this format is defined in Section 3 of the Annotation Protocol [annotation-protocol] as application/ld+json;profile="http://www.w3.org/ns/anno.jsonld".
As explained in [w3/annotation-mode#serialization-of-the-model](https://www.w3.org/TR/annotation-model/#serialization-of-the-model)